packages/asset-swapper/src/utils/market_operation_utils/types.ts

```diff
@@ -195,6 +196,7 @@ export interface GetMarketOrdersOpts {
      */
     allowFallback: boolean;
     rfqt?: GetMarketOrdersRfqtOpts;
+    quoteReporter?: QuoteReporter;
```

Comment:
Is it bad to make this required? We merge in defaults anyway

Reply:
I think this relates to the convo about the null quote reporter pattern: https://github.com/0xProject/0x-monorepo/pull/2606#discussion_r448727337